pkgs/by-name/de/deepcool-digital-linux/package.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  libudev-zero,
+}:
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "deepcool-digital-linux";
+  version = "0.8.3-alpha";
+
+  src = fetchFromGitHub {
+    owner = "Nortank12";
+    repo = "deepcool-digital-linux";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-Whmjd6NCOUkE7hM3FaN7grMwcC/suL7AJDVSgnZSKzM=";
+  };
+
+  cargoHash = "sha256-K1pEbUyENPUS4QK0lztWmw8ov1fGrx8KHdODmSByfek=";
+
+  buildInputs = [ libudev-zero ];
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
```

can be tested with `nix run nixpkgs#nix-update -- deepcool-digital-linux`
```suggestion

  passthru.updateScript = nix-update-script { };

```